bin/load-queued-items.ts

```diff
@@ -118,31 +102,51 @@ async function moveQueuedItems(feedId, baseTime) {
 
   let i = 1;
   while (queuedItem) {
-    const curatedRecId = await insertIntoCuratedFeedItem(
-      queuedItem,
-      i * 3600 + baseTime
-    );
-    await updateQueuedItem(queuedItem['queued_id']);
-    await insertTileSource(curatedRecId);
-    console.log(JSON.stringify(queuedItem));
+    const trx = await dbClient.transaction();
```

Comment:
I'm not sure if you can reuse this transaction object in a loop once you've committed or rolled back in line 114/117